### PR TITLE
Renamed Dropdown classes and components #DS-1248

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-react/src/components/Dropdown/__tests__/Dropdown.test.tsx
Comment (literat) [inline packages/web-react/src/components/Dropdown/__tests__/Dropdown.test.tsx:1]: Just to be sure: Do we apply our new rule about test refactoring to this case?

**Review by literat — APPROVED** [review]
LGTM 👍

**Review by curdaj — APPROVED** [review]

**Comment by literat** [comment]
@curdaj thanks for the test suggestion. @crishpeen can we apply them, please? 🙏

File: packages/web-react/src/components/Dropdown/__tests__/Dropdown.test.tsx
Comment (crishpeen) [reply packages/web-react/src/components/Dropdown/__tests__/Dropdown.test.tsx:1]: we do, now.

**Pull request merged**